app.rb

```diff
@@ -0,0 +1,120 @@
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+
+class App
+  attr_accessor :people, :rentals, :books
+
+  def initialize
+    @people = []
+    @rentals = []
+    @books = []
+  end
+
+  def display_book_list
+    if @books.empty?
+      puts 'The book list is empty.'
+    else
+      @books.each_with_index do |book, index|
+        puts "#{index} - Title: #{book.title.capitalize}, Author: #{book.author.capitalize}"
+      end
+    end
+  end
+
+  def display_people_list
+    if @people.empty?
+      puts 'The person list is empty.'
+    else
+      @people.each_with_index do |person, index|
+        person_type = person.instance_of?(Teacher) ? '[Teacher]' : '[Student]'
+        puts "#{index} #{person_type} Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    end
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [input the number]:'
+    type = gets.chomp
+    case type
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Invalid selection'
+    end
+  end
+
+  def create_teacher
+    puts 'Age:'
+    age = gets.chomp
+    puts 'Name:'
+    name = gets.chomp
+    puts 'Specialization:'
+    specialization = gets.chomp
+    parent_permission = true
+    @people.push Teacher.new(age, specialization, parent_permission: parent_permission, name: name)
+    puts 'Person created successfully'
+  end
+
+  def create_student
+    puts 'Name:'
+    name = gets.chomp
+    puts 'Age:'
+    age = gets.chomp
+    puts 'Has parent permission? [y/n]:'
+    permission = gets.chomp.downcase
+    parent_permission = permission == 'y'
+    puts 'Classroom:'
+    classroom = gets.chomp
+    @people.push Student.new(age, classroom, parent_permission: parent_permission, name: name)
+    puts 'Person created successfully'
+  end
```

Comment:
- [ ] Good job for creating a student 👍👍 However, I noticed that the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) does not mention anything about the student's classroom. Please make sure to follow the [project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md) and fix this ✨⛏

  ![image](https://github.com/rugwizangoga/School-Library-app/assets/106119105/5fd2fae8-f704-4c6a-ab2e-2627c807ee43)